### DISCUSSION TITLE
please make it work on android 9

### DISCUSSION THREAD

**Discussion by Ajjajaja12** [skyline-emu/skyline#2027] [Q&A] [answered]
please make it work on android 9

**Comment by AndresNorm** [comment]
It will not.
Upgrade your Phone to Android 10
Or buy a new one.

**Comment by griffi-gh** [comment, marked as answer]
If your phone runs (still) runs android 9 it's *probably* not good enough for skyline

**Reply by Ajjajaja12** [reply]
I have a snapdragon 845 and 6 RAM

**Reply by griffi-gh** [reply]
then flashing a custom ROM is the only option for you for now. It's unlikely that support for outdated android versions will ever get implemented.

**Reply by Ajjajaja12** [reply]
I don't have a pc
